CHANGELOG.md

```diff
@@ -6,15 +6,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 - Update tests to cover `resend_code` feature in `passwordless` and `thirdpartypasswordless` recipe.
-
 - Update usermetadata tests to ensure that utf8 chars are supported.
-
-# Fixes
-- Fixed handling of unicode characters in usermetadata (and emails, roles, session/access token payload data)
```

Comment:
This isn't required since `Update usermetadata tests to ensure that utf8 chars are supported`  refers to the same thing.